maintenance/ManageInactiveWikis.php

```diff
@@ -48,8 +48,9 @@ public function execute(): void {
 			->fetchFieldValues();
 
 		foreach ( $wikis as $wiki ) {
-			$remoteWiki = $remoteWikiFactory->newInstance( $wiki );
-			$inactiveDays = (int)$this->getConfig()->get( ConfigNames::StateDays )['inactive'];
+			$dbname = $wiki->wiki_dbname;
+			$remoteWiki = $remoteWikiFactory->newInstance( $dbname );
+			$inactiveDays = (int)$this->getConfig()->get( ConfigNames::StateDays )['no-edits']['inactive'];
```

Comment:
I know it's already here but 1) why hardcover 'no-edits' and 2) move this out of the loop no need to recreate the variable every time.

Reply:
I did this myself